{"name": "test_verify_error_notification_with_invalid_password", "status": "passed", "start": 1709468001421, "stop": 1709468001848, "uuid": "9724f8f6-2e85-42bb-a99d-f05a4145ba50", "historyId": "11618e94f4ee8debe8d5587e406202fb", "testCaseId": "11618e94f4ee8debe8d5587e406202fb", "fullName": "Tests.test_login.TestSearch#test_verify_error_notification_with_invalid_password", "labels": [{"name": "tag", "value": "@pytest.mark.usefixtures('setup_and_teardown')"}, {"name": "parentSuite", "value": "Tests"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestSearch"}, {"name": "host", "value": "DESKTOP-UEVTOM8"}, {"name": "thread", "value": "26908-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "Tests.test_login"}]}